Syntaxes/debian-changelog.sublime-syntax

```diff
@@ -9,30 +9,84 @@ scope: source.debian.changelog
 
 contexts:
   main:
-    - include: changelog-header
-    - include: changelog-footer
-    - include: content-close-bug
-    - include: content-code-span
+    - include: header
+
+  header:
+    - match: ^[\w-]+
+      scope: entity.name.class.debian.changelog
+      push: [body, header-annotations]
 
-  changelog-header:
-    - match: (.+) [(](.+)[)] (.+); (.+)
+  header-annotations:
+    - match: \(
+      scope: punctuation.section.group.begin.debian.changelog
+      push:
+        - meta_scope: meta.version-number.debian.changelog
+                      meta.parens.debian.changelog
+        - include: debian-common.sublime-syntax#semantic-version
+        - match: \)
+          scope: punctuation.section.group.end.debian.changelog
+          pop: true
+        - include: debian-common.sublime-syntax#pop-eol
+    - match: \b(?:un)?stable
+      scope: constant.language.debian.changelog
```

Comment:
We're missing a few possible matches: [experimental](https://sources.debian.org/src/neofetch/6.0.0-2/debian/changelog/#L68), [buster](https://sources.debian.org/src/backuppc/3.3.2-2+deb10u1/debian/changelog/#L1) (and all Debian version names).
Could we possibly just match everything between the closing parenthesis of the version number and the semicolon?

Reply:
I think we should consider matching explicitly. I'll look into whether there's a good way to do this in a way that doesn't require a list of Debian versions in more than one file.

Reply:
Sounds good for me!